docs/concepts/macros/sqlmesh_macros.md

```diff
@@ -901,6 +901,40 @@ FROM rides
 GROUP BY 1
 ```
 
+### @DEDUPLICATE
+
+`@DEDUPLICATE` is used to deduplicate rows in a table based on the specified partition and order columns with a window function.
+
+It supports the following arguments, in this order:
+
+- `relation`: The table or CTE name to deduplicate
+- `partition_by`: column names, or expressions to use to identify a window of rows out of which to select one as the deduplicated row
+- `order_by`: A list of strings representing the ORDER BY clause, null values in columns ordered first
```

Comment:
Ahhh I did misunderstand what you were saying. I was trying to avoid giving more order by configs to the user to avoid bloat, but in the error message it's good to include them. Updated my error messages and tests to include nulls ordering